plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java

```diff
@@ -1860,11 +1863,13 @@ public Answer createTemplateFromSnapshot(CopyCommand cmd) {
                 }
                 vmMo = clonedVm;
             }
+            s_logger.debug(String.format("Exporting volume to export path [%s], with VM config [%s].", exportPath, _gson.toJson(vmMo)));
```

Comment:
noticed this log multiple times, is it possible to move the log to _exportVm()_ method in VirtualMachineMO class